app.py

```diff
@@ -41,7 +48,7 @@ def init_db():
 # make sure you have Content-Type: application/json in the request header
 # payload JSON should have the following fields:
 # 'title' (required): string of note title
-# 'content' (optional): string of note content
+# 'content' (optional): list of strings for content
```

Comment:
why was this added?

Reply:
This was a mistake lol. I can change this